src/layouts/episode.njk

```diff
@@ -1,61 +1,42 @@
 ---
 layout: main.njk
+scripts: ['player']
 ---
 
-{# <Helmet>
-    <meta name="twitter:card" content="summary" />
-    <meta
-        name="twitter:creator"
-        content={data.site.siteMetadata.twitter}
-    />
-    <meta property="og:url" content={location.href} />
-    <meta
-        property="og:title"
-        content={`#${post.frontmatter.number}: ${post.frontmatter.title}`}
-    />
-    <meta
-        property="og:description"
-        content={post.frontmatter.summary}
-    />
-    <meta
-        name="twitter:image"
-        content={`${data.site.siteMetadata.siteURL}${Logo}`}
-    />
-    <meta property="og:image" content={Logo} />
-</Helmet> #}
-
 
 <div class="post__header">
     <h1 class="title--page">{{title}}</h1>
     <span class="excerpt__subtitle">
-        <time datetime="{{date | datefull }}" itemprop="datePublished">{{date | date}}</time>
+        <span class="excerpt__subtitle">Episode #{{number}}:</span> <time datetime="{{date | datefull }}" itemprop="datePublished">{{date | date}}</time>
     </span>
 </div>
 <div class="post__content">
 
-        {# <span class="visuallyhidden">Published on </span>
-        <time datetime="{{date | datefull }}" itemprop="datePublished">{{date | date}}</time>
+    <div class="meta">
         <span class="visuallyhidden">Modified on <time datetime="{{date | datefull }}" itemprop="dateModified">{{date | date}}</time></span>
         <span class="visuallyhidden" itemprop="author" itemscope="" itemtype="http://schema.org/Person">
             <span itemprop="name"><a href="https://plus.google.com/111879829548102811838" itemprop="url" rel="author">Tom Hazledine</a></span>
         </span>
         <span class="visuallyhidden" itemprop="publisher" itemscope="" itemtype="http://schema.org/Organization">
-            <meta itemprop="name" content="Tom Hazledine" />
+            <meta itemprop="name" content="{{global.siteTitle}}" />
             <div itemprop="logo" itemScope itemtype="https://schema.org/ImageObject">
```

Comment:
What does this `itemtype` do?

Reply:
Part of the structured schema data - really helpful for SEO etc. https://schema.org/

Reply:
👍